### DISCUSSION TITLE
这个注释是不是作者打错了？（星号很多的哪里）

### DISCUSSION THREAD

**Discussion by fj-270** [fangwei123456/SpikingFlow#644] [General]
# 仿真周期
T = 20 # 把仿真时间T设置为20
size = 10 # 生成的数据大小为10

# 随机生成一个大小为[size]的数据x
x = torch.rand([size])
print(f"x = {x}")

# 配置编码器
encoder = encoding.PoissonEncoder()
x_encoded_size = [T, size] # 编码后的数据大小
x_encoded = torch.zeros(x_encoded_size) # 编码后的数据**********************************************************************************
# 开始按照时间逐步编码
for t in range(T):
    x_encoded[t] = encoder(x)
print(f"x_encoded = {x_encoded}")

num = x_encoded.sum(dim=0) # 将编码后的脉冲(1)按照维度0（时间维度）求和
rate = num / T # 得到编码后的概率
print(f"rate = {rate}")
